src/main/resources/templates/db_scripts/common/createRecordsViewFunction.sql

```diff
@@ -0,0 +1,324 @@
+-- Creates auxiliary functions
```

Comment:
This script won't be executed unless you specify it in `schema.json`

Reply:
Good point, I'll add script entry there. Only question related to specification of the version, can we set next release version which is expected to be an 19.3.2 ?

Reply:
> Only question related to specification of the version, can we set next release version which is expected to be an 19.3.2 ?

Why would we want to use a bug fix version for this? As I understand it, this change isn't intended to be back ported and so will be released from the mainline as either a new minor or major version.

Reply:
Do you mean, that we need to specify 19.4.0 version for script here?